spaceapi_server/src/lib.rs

```diff
@@ -13,9 +14,10 @@ pub mod sensors;
 use std::net::Ipv4Addr;
 
 use rustc_serialize::json::{Json, ToJson};
-use iron::{Request, Response, IronResult, Iron, Set};
+use iron::prelude::*;
```

Comment:
I'm still not fond of star imports. It's very implicit and it may shadow other imports. What's the problem with the explicit import list?